### ISSUE TITLE
BUG: autoCreatedAt autoUpdatedAt

### ISSUE THREAD

**Issue by anasyb** [balderdashy/sails-mongo#344]
When setting autoUpdatedAt and autoCreatedAt to false, then doing some updates that involves associations, waterline is throwing a mongo db error. Just commenting them out resolves the issue.

```
Error (E_UNKNOWN) :: Encountered an unexpected error
MongoError: '$set' is empty. You must specify a field like so: {$set: {<field>: ...}}
    at Function.MongoError.create (/server/node_modules/sails-mongo/node_modules/mongodb/node_modules/mongodb-core/lib/error.js:31:11)
    at toError (/server/node_modules/sails-mongo/node_modules/mongodb/lib/utils.js:114:22)
    at /server/node_modules/sails-mongo/node_modules/mongodb/lib/collection.js:923:67
    at /server/node_modules/sails-mongo/node_modules/mongodb/node_modules/mongodb-core/lib/topologies/server.js:756:13
    at bound (domain.js:250:14)
    at runBound (domain.js:263:12)
    at Callbacks.emit (/server/node_modules/sails-mongo/node_modules/mongodb/node_modules/mongodb-core/lib/topologies/server.js:95:3)
    at null.messageHandler (/server/node_modules/sails-mongo/node_modules/mongodb/node_modules/mongodb-core/lib/topologies/server.js:243:23)
    at Socket.<anonymous> (/server/node_modules/sails-mongo/node_modules/mongodb/node_modules/mongodb-core/lib/connection/connection.js:262:22)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at readableAddChunk (_stream_readable.js:146:16)
    at Socket.Readable.push (_stream_readable.js:110:10)
    at TCP.onread (net.js:523:20)

Details:  MongoError: '$set' is empty. You must specify a field like so: {$set: {<field>: ...}}

```

**Comment by sailsbot** [comment]
Thanks for posting, @anasyb.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
